src/main/java/software/amazon/encryption/s3/S3AsyncEncryptionClient.java

```diff
@@ -286,6 +288,11 @@ public static class Builder implements AwsAsyncClientBuilder<Builder, S3AsyncEnc
         private ClientAsyncConfiguration _clientAsyncConfiguration = null;
         private SdkAsyncHttpClient _sdkAsyncHttpClient = null;
         private SdkAsyncHttpClient.Builder _sdkAsyncHttpClientBuilder = null;
+        private S3Configuration _serviceConfiguration = null;
+        private Boolean _accelerate = null;
```

Comment:
Curious why `Boolean` and not `boolean`? Other fields in these classes are `boolean`s

Reply:
The SDK uses `Boolean` and I prefer to match the SDK for its configuration.

Reply:
Is that preferable to maintaining type consistency within our own client?
Right now our client uses both `boolean` and `Boolean`.
I'd prefer that our own client is consistent within itself (i.e. exclusively uses one or the other),
over having some fields be consistent with what the SDK uses.
That said, I don't understand the impact of mixing these types, but I imagine it's low and this is almost certainly just a pedantic conversation. If it is, just let me know and I'll drop this thread.

Reply:
Mixing them is fine, IMO, since they have different semantics. `boolean` cannot be `null`, whereas `Boolean` can. I prefer using `boolean` when possible so as to avoid having to handle `null`. There really isn't any advantage to using `Boolean` internally, but there is here as it maintains compatibility with the SDK.